src/main/webapp/electionlist.html

```diff
@@ -94,10 +94,16 @@ <h3>Elections coming up in {{state}}</h3>
             {{#each stateElections}}
             <li class="list-group-item election-item">
               <div class="election-quick-details">
-                <p>{{this.name}}</p>
+                <div id="state-election-name">
+                  <p>{{this.name}}</p>
+                </div>
                 <p><em>{{formatDate this.date}}</em></p>
               </div>
-              <div class="learn-more-button">Learn more</div>
+              <a href="#" 
+                 onclick="redirectToStateElectionPage({{{this.ID}}});return false;" 
+                 id="election-choice-js">
```

Comment:
I don't think you can/should have multiple id tags of the same id on a given page.

Reply:
This was how I reasoned we could access the ID from within the handlebars script; is there another way you recommend accomplishing this?

Reply:
I was referring to 
```
id="election-choice-js"
```

Can't you just change this to `class="..."`?

Reply:
oh I see -- will do!